### DISCUSSION TITLE
crazyflie commander with Vicon MoCap

### DISCUSSION THREAD

**Discussion by ntukenmez3** [bitcraze/discussions#1530] [Q&A] [answered]
Hello,
We use Vicon MoCap system for positioning.
We've seen that ` cf.commander.send_setpoint(roll, pitch, yaw rate, thrust)` is not working with Vicon.
How can we fix it? We couldn't get the issue. `cf.commander.send_position_setpoint()` is working well but the other commander function is not working.

**Comment by knmcguire** [comment, marked as answer]
I've just converted this to a discussion.

The send_setpoint doesn't require any positioning it all so it should work always, mocap or not, as it is driven by the internal IMU.

What is probably going on is that the crazyflie is locked. If you send it one send_setpoint command with zero thrust which will unlock it. You can find that information in the api documentation here: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/python_api/#sending-control-setpoints-with-the-commander-framework

**Reply by ntukenmez3** [reply]
It's fixed. Thank you.
